dashboard/app/graphs.go

```diff
@@ -518,19 +502,23 @@ func handleGraphCrashes(c context.Context, w http.ResponseWriter, r *http.Reques
 			return err
 		}
 	} else {
-		data.Table = createCrashesTable(c, hdr.Namespace, data.TableDays.Val, bugs)
+		data.Table = createCrashesTable(c, hdr.Namespace, data.TableDays.Val, bugs, accessLevel(c, r))
 	}
 	return serveTemplate(w, "graph_crashes.html", data)
 }
 
-func createCrashesTable(c context.Context, ns string, days int, bugs []*Bug) *uiCrashPageTable {
+func createCrashesTable(c context.Context, ns string, days int, bugs []*Bug,
+	accessLevel AccessLevel) *uiCrashPageTable {
 	const dayDuration = 24 * time.Hour
 	startDay := timeNow(c).Add(time.Duration(-days+1) * dayDuration)
 	table := &uiCrashPageTable{
 		Title: fmt.Sprintf("Top crashers of the last %d day(s)", days),
 	}
 	totalCount := 0
 	for _, bug := range bugs {
+		if accessLevel < bug.sanitizeAccess(accessLevel) {
```

Comment:
Maybe filter right after the line 489 instead?

Reply:
Done